docs/contributing/contributing.md

```diff
@@ -1,20 +1,23 @@
 ---
-slug: how-to-contribute
+slug: contributing
 description: "Semgrep is LGPL-licensed and contributions are welcome. Get started by filing an issue, fixing a bug, contributing rules to the registry, adding a feature, or updating the docs. You can also contribute by helping others in the r2c Community Slack!"
 ---
 
-# How to contribute
+# Contributing
 
-Semgrep is LGPL-licensed and contributions are welcome.
+Your contributions are welcome!
 
-To start contributing, first please make sure you read and agree with the [Contributor Covenant Code of Conduct](https://github.com/returntocorp/semgrep/blob/develop/CODE_OF_CONDUCT.md).
-Then check out a few ways you can get involved:
+To contribute, read and agree with the [Contributor Covenant Code of Conduct](https://github.com/returntocorp/semgrep/blob/develop/CODE_OF_CONDUCT.md).
 
-- [File an issue](https://github.com/returntocorp/semgrep/issues/new/choose)
-- Fix a bug — pick from the [good first issues](https://github.com/returntocorp/semgrep/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) or work on any of the [currently open bugs](https://github.com/returntocorp/semgrep/issues?q=is%3Aopen+is%3Aissue+label%3Abug)
-- Contribute rules — add rules to the [semgrep-rules repository](https://github.com/returntocorp/semgrep-rules)
-- Add a feature — see the [enhancement issues](https://github.com/returntocorp/semgrep/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement) for inspiration
-- Update the [docs](https://github.com/returntocorp/semgrep-docs)
-- Help each other in the [r2c Community Slack](https://r2c.dev/slack)
+There are various ways in which you may contribute:
 
-Please see the [contribution guidelines](contributing-code.md) for info about the development workflow, testing, and making PRs. For a higher-level view of Semgrep’s design principles, see the [Semgrep CLI philosophy](../semgrep-philosophy/).
+| Contribution | Where to contribute |
+|-----------------------|-----------------------|
+| File a Semgrep issue. | See the [Semgrep GitHub repository](https://github.com/returntocorp/semgrep/issues/new/choose). |
+| Fix an issue. | Pick a bug from the [good first issues](https://github.com/returntocorp/semgrep/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) or work on any of the [currently open bugs](https://github.com/returntocorp/semgrep/issues?q=is%3Aopen+is%3Aissue+label%3Abug). |
+| Contribute to the rules repository. | Add new rules through Semgrep App or GitHub. See [Contributing to Semgrep rules](.../../contributing-rules.md). |
+| Add a new feature. | See the [enhancement issues](https://github.com/returntocorp/semgrep/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement) for inspiration. |
+| Update the documentation you are reading right now! | Create a PR or an issue in the [Documentation repository](https://github.com/returntocorp/semgrep-docs). |
+| Help others in the community. | Check [r2c Community Slack](https://r2c.dev/slack). |
```

Comment:
@s-santillan let me know if you know a better table styling. I know that there are just two things, so this could be a description list, but maybe a table is better to really show where a user can contribute?

Reply:
The tables look good when rendered :) You can use tabularize if the source markdown bothers you 😅

Reply:
Will rather keep it as it was. But thank you! This will certainly proof useful later.
![Screenshot 2022-05-23 at 15 51 37](https://user-images.githubusercontent.com/26669307/169843887-7338e195-a842-4c31-aa57-7d682bee1038.png)